react-app-rewired/index.ts

```diff
@@ -391,6 +391,10 @@ const reactAppRewireConfig = {
       '/node_modules/(?!(@wagmi)/)',
       '^.+\\.module\\.(css|sass|scss)$',
     ]
+    // allows jest to not crash when our deps import different versions of axios
+    config.moduleNameMapper = {
+      '^axios$': require.resolve('axios'),
+    }
```

Comment:
this was necessary because pulling lifi `getTradeQuote` into redux api meant that 30 unit tests were failing due to lifi sdk using a version of `axios` that isnt compatible with jest. This ensures axios is resolved to our intalled version while running tests.